### PR TITLE
Facility CSV Download for Admins

### PR DESCRIPTION
**Pull request by rdeepak2002**
- Ability to download facility data as CSV file in admin portal page
- Fix React CSS error with classnames not being camel-case
- Remove TODO where API URL was being printed to console
- Ability to submit login and register page credentials by pressing enter key

### CODE REVIEW

File: src/adminportal/index.tsx [hunk 1]
```diff
@@ -8,16 +8,56 @@ import firebaseApp from '../firebase';
...
+    useEffect(() => {
+        async function fetchFacilityData() {
+            try {
+                const newFacilityData: Array<Array<string>> = [CSV_FIELDS];
+                const querySnapshot = await getDocs(query(collection(db, "facility")));
+                querySnapshot.forEach((doc) => {
+                    const facility = doc.data();
+                    const facilityDataArr: Array<string> = [];
+                    for (const field of CSV_FIELDS) {
+                        facilityDataArr.push(facility[field] ? `${makeStringCSVCompliant(JSON.stringify(facility[field]))}` : "");
```
Comment (Skyline-9) [inline src/adminportal/index.tsx:36]: refactor `makeStringCSVCompliant to handle` null/empty

File: src/adminportal/styles.module.css [hunk 2]
```diff
@@ -124,4 +124,21 @@
...
-}
\ No newline at end of file
+}
+
+.downloadBtn:disabled, .downloadBtn[disabled] {
+    border: 1px solid #a59de1;
+    background: #a59de1;
+}
+
+a,button {
```
Comment (Skyline-9) [inline src/adminportal/styles.module.css:134]: refactor class name

File: src/adminportal/index.tsx [hunk 3]
```diff
@@ -70,10 +110,14 @@ function AdminPortalPage() {
...
                 </div>
             </div>
-            <button className={styles.downloadBtn} onClick={() => {
-                alert('Data Downloaded')
-            }}>Download Data
-            </button>
+            {
+                (facilityData && facilityData.length > 0) ?
+                <CSVLink data={facilityData} filename={'facilities.csv'} className={styles.downloadBtn}>
+                    <span>Download Facilities CSV File</span>
```
Comment (Skyline-9) [inline src/adminportal/index.tsx:116]: Download as CSV File is too verbose for a button, we should shorten this so text doesn't overflow on mobile

**Review by Skyline-9 — CHANGES_REQUESTED** [review]

File: src/adminportal/index.tsx (hunk 3 above)
Comment (rdeepak2002) [reply src/adminportal/index.tsx:116]: Sounds good. Renaming to "Download CSV."

File: src/adminportal/index.tsx (hunk 1 above)
Comment (rdeepak2002) [reply src/adminportal/index.tsx:36]: Done.

**Review by Skyline-9 — APPROVED** [review]
Looks good to me!

**Pull request merged**